onnxruntime/test/providers/cpu/tensor/resize_op_test.cc

```diff
@@ -1179,5 +1180,27 @@ TEST(UpsampleOpTest, ResizeOpNearestNoScaleTest_Ver10) {
   test.Run();
 }
 
+TEST(ResizeOpTest, ResizeOp_MissingRoiAndMissingScalesOptionalInputs) {
+  OpTester test("Resize", 13);
```

Comment:
There are separate version 10 tests (the schema was vastly different in opset-10)